ZAPD/Globals.cpp

```diff
@@ -176,11 +177,43 @@ void Globals::AddSegment(int32_t segment, ZFile* file)
 	if (std::find(segments.begin(), segments.end(), segment) == segments.end())
 		segments.push_back(segment);
 
-	segmentRefs[segment] = file->GetXmlFilePath();
+	segmentRefs[segment] = file->GetXmlFilePath().string();
 	segmentRefFiles[segment] = file;
 }
 
 bool Globals::HasSegment(int32_t segment)
 {
 	return std::find(segments.begin(), segments.end(), segment) != segments.end();
 }
+
+std::map<std::string, ExporterSet*>* Globals::GetExporterMap()
+{
+	static std::map<std::string, ExporterSet*> exporters;
+	return &exporters;
+}
+
+void Globals::AddExporter(std::string exporterName, ExporterSet* exporterSet)
+{
+	std::map<std::string, ExporterSet*>* exporters = GetExporterMap();
+	(*exporters)[exporterName] = exporterSet;
+}
+
+ZResourceExporter* Globals::GetExporter(ZResourceType resType)
+{
+	auto exporters = *GetExporterMap();
```

Comment:
Doing this would make a copy of the whole Exporter Map, which would be a waste of resources imo.
What about using the pointer instead? 
```suggestion
	auto exporters = GetExporterMap();
```
If you don't want to let the user modify the original, you could either return a `const` pointer or a `const` reference.